Tasks/AzureFunctionAppV1/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -7,22 +7,29 @@ import { KUDU_DEPLOYMENT_CONSTANTS } from 'azure-pipelines-tasks-azurermdeployco
 
 export class KuduServiceManagementClient {
     private _scmUri;
-    private _accesssToken: string;
+    private _accessToken: string;
     private _cookie: string[];
+    private _scmAccessCheck : boolean;
 
-    constructor(scmUri: string, accessToken: string) {
-        this._accesssToken = accessToken;
+    constructor(scmUri: string, accessToken: string, scmAccessCheck: boolean) {
+        this._accessToken = accessToken;
         this._scmUri = scmUri;
+        this._scmAccessCheck = scmAccessCheck;
     }
 
     public async beginRequest(request: webClient.WebRequest, reqOptions?: webClient.WebRequestOptions, contentType?: string): Promise<webClient.WebResponse> {
         request.headers = request.headers || {};
-        request.headers["Authorization"] = "Basic " + this._accesssToken;
-        request.headers['Content-Type'] = contentType || 'application/json; charset=utf-8';
-        
-        if(!!this._cookie) {
-            tl.debug(`setting affinity cookie ${JSON.stringify(this._cookie)}`);
-            request.headers['Cookie'] = this._cookie;
```

Comment:
Did we intent to remove the affinity cookie here ?

Reply:
I think we didn't intend to, I was just trying to streamline the code. This can be put back.